UnityProject/Assets/Scripts/Objects/Machines/QuantumPad.cs

```diff
@@ -184,6 +191,12 @@ public void ServerDetectObjectsOnTile()
 			{
 				TransportUtility.TransportObjectAndPulled(item, travelCoord);
 				somethingTeleported = true;
+
+				if(item.GetComponent<Bread>() == true)
```

Comment:
Use item traits instead, example:

Validations.HasItemTrait(this.gameObject, CommonTraits.Instance.SpillOnThrow)

you'll need to create the new itemtrait, theyre kept in resources/scriptableobjects

and then assign the item trait to the breads ItemAttributesV2 component